packages/@expo/fingerprint/cli/src/commands/diffFingerprints.ts

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
```

Comment:
```suggestion
```

i know both @expo/cli and expo-updates has the shebang, but it's not necessary because this file is be imported